File: 1546.c
#include <stdio.h>
int main(void){
    int n;
    double a, b = 0, sum = 0;
    scanf("%d",&n);
    double s[n];
    for(int i = 0; i < n; i++){
        scanf("%lf",&a);
        s[i] = a;
    }
    for(int j = 0; j < n; j++){
        if(s[j] > b){
            b = s[j];
        }
    }
    for(int k = 0; k < n; k++){
        s[k] = s[k]/b*100;
        sum += s[k];
    }
    printf("%.2f\n",sum/n);
    return 0;
}

File: 1546.java
import java.util.Scanner;

class Main {
    public static void main(String[] args) {
        Scanner s = new Scanner(System.in);
        int n = s.nextInt();
        double b = 0, sum = 0;
        double[] array = new double[n];
        for (int i = 0; i < n; i++) {
            double a = s.nextDouble();
            array[i] = a;
        }
        for (int j = 0; j < n; j++) {
            if (array[j] > b) {
                b = array[j];
            }
        }
        for (int k = 0; k < n; k++) {
            array[k] = array[k] / b * 100;
            sum += array[k];
        }
        System.out.printf("%.2f\n", sum / n);
    }
}